### ISSUE TITLE
User input prompt from REPL instead of GUI?

### ISSUE THREAD

**Issue by ericdevg** [microsoft/poshtools#366]
In order to support Console.ReadKey(), it would make more sense to prompt user input through the REPL window instead of pop up a input dialog for it.

**Comment by ASayre** [comment]
Agreed, we should [eventually] support user Input from CmdLine instead of a gui in general.

However, is it simple to have a quick prompt for ReadKey at this point in time, but so we don't block people who have scripts with ReadyKey in them?

**Comment by ericdevg** [comment]
For now ReadKey is crashing the console app itself due to "no console at all", so it is technically not blocking anything from continuing using posh tools. so we'd address it when we work on REPL prompt.
